### DISCUSSION TITLE
Builds doesn't apply on 2.x

### DISCUSSION THREAD

**Discussion by jordanyoong1** [cangzhang/ChampR#279] [General]
Nothing happens after pressing rocket button, runes appy fine though

**Comment by Caponia** [comment]
先用v1版本的吧，v2版本的出装会下载到别的文件夹，需要自己剪切。

**Comment by cangzhang** [comment]
Just check the console output, we'll show the apply progress indicator later
![image](https://github.com/cangzhang/champ-r/assets/1357073/c71ef859-1e68-4b5e-8f5b-9c3a3a863a98)

**Reply by jordanyoong1** [reply]
按了火箭后之后完全没反应, 刚刚试了一下v2.0.2-b2没问题
![Animation](https://github.com/cangzhang/champ-r/assets/40160073/f0eb0a17-bfdf-448f-b5f3-efd33c09dadb)

**Comment by cangzhang** [comment]
https://t.me/champ_r/98

> 各位如果遇到连接不到客户端的问题，请检查自己的操作系统是否是 win10，以及 lol 安装路径是否包含中文。
> 
> 如果是 win10 且包含中文的话，将安装路径改成英文则可解决该问题。
> 
> 如果是 win11，中文路径应该没有问题（本机测试）。
> 
> 感谢 sfzhang 的帮助调查👍，如果有其他问题请各位继续反馈

**Comment by cangzhang** [comment]
https://github.com/cangzhang/champ-r/releases/tag/v2.0.2-b5
